src/rgw/CMakeLists.txt

```diff
@@ -210,6 +210,7 @@ target_link_libraries(rgw_common
     cls_timeindex_client
     cls_user_client
     cls_version_client
+    dfs
```

Comment:
I think we do not need this here, because it is included when `WITH_RADOSGW_DAOS` is added,

Reply:
You're exactly right! Updated.